card/src/main/java/com/adyen/checkout/card/BinLookupData.kt

```diff
@@ -0,0 +1,16 @@
+package com.adyen.checkout.card
+
+/**
+ * Class that contains data about the entered card number.
+ *
+ * @param brand The detected brand.
+ * @param paymentMethodVariant The type of payment method. See the
+ * [docs](https://docs.adyen.com/development-resources/paymentmethodvariant/) for more information.
+ * @param isReliable Indicates whether the data is reliable. If true, the card number is checked with Adyen systems.
+ * If false, the data is provided by a local detection algorithm.
+ */
+data class BinLookupData(
```

Comment:
I wonder if we should call this something else, bin lookup is an internal naming and might not be obvious. What about "card details"? Can we sync with other platforms and product?

Reply:
I do like "card details" more as well. The callback is already called onBinLookup though, because it's name they use for web. You think we should change it?

Reply:
Yeah I would say card details is clearer but if iOS and web use bin lookup then we could go for either 🤷

Reply:
On iOS it's called `didChangeCardBrand`. So we can choose for consistency with one of the platforms or we do our own naming